telemetry_core/src/test/java/com/newrelic/telemetry/EventBatchSenderFactoryTest.java

```diff
@@ -0,0 +1,29 @@
+package com.newrelic.telemetry;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.newrelic.telemetry.http.HttpPoster;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class EventBatchSenderFactoryTest {
+
+  EventBatchSenderFactory factory;
+
+  @BeforeEach
+  void setup() {
+    HttpPoster poster = (url, headers, body, mediaType) -> null;
+    factory = () -> poster;
```

Comment:
can we just put this in the declaration, since it's stateless?

Reply:
Good idea!